internal/controllers/aggregation/composition.go

```diff
@@ -108,9 +103,6 @@ func (c *compositionController) aggregate(synth *apiv1.Synthesizer, comp *apiv1.
 	if comp.Status.CurrentSynthesis.Ready != nil {
 		copy.Status = "Ready"
 	}
-	if comp.Status.PendingResynthesis != nil {
-		copy.Status = "WaitingForCooldown"
```

Comment:
This was too chatty anyway - generated tons of writes just to update a relatively short-lived and unimportant status message. In any case, `PendingResynthesis ` is going away ⚡